cmd/livepeer/livepeer.go

```diff
@@ -102,6 +102,7 @@ func main() {
 	faceValue := flag.Float64("faceValue", 0, "The faceValue to expect in PM tickets, denominated in ETH (e.g. 0.3)")
 	winProb := flag.Float64("winProb", 0, "The win probability to expect in PM tickets, as a percent float between 0 and 100 (e.g. 5.3)")
 	maxSessions := flag.Int("maxSessions", 10, "Orchestrator only. Maximum number of concurrent transcoding sessions")
+	webhookURL := flag.String("webhookUrl", "", "Authentication webhook URL")
```

Comment:
Maybe the description could be "RTMP authentication webhook URL". Should we also call it something like `authUrl` so we don't have to rename things if/when we add more hooks later on?

Reply:
Renamed